### PR TITLE
App doc

### LINKED ISSUE #4: Add docs for app (bike messenger)
- [x] Remove "Applications smartphone" from the sidebar
- [x] Create new doc section for app in the "courier" folder
   - Explain how to complete a task
   - Explain how to take a proof of delivery (photo/signature)

### PR DESCRIPTION
**Pull request by arnomasson**
Fixes #4

### CODE REVIEW

File: fr/coursier/tache.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+lang: fr
+role: coursier
+title: Guide du coursier
+ref: coursier-home
+---
+
+## Compléter une tâche :
+
+Dès qu'une nouvelle tâche vous est assigné, vous recevez une notification sur votre téléphone.
```
Comment (alexsegura) [inline fr/coursier/tache.md]: `assigné` → `assignée`

File: fr/coursier/tache.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Pour accéder à toutes les informations concernant cette tâches, il existe deux possibilités :
```
Comment (alexsegura) [inline fr/coursier/tache.md]: `cette tâches` → `cette tâche`

File: fr/coursier/tache.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Soit en cliquant sur l'icône ![Position tâche](/assets/images/geoloc_icone.png) correspondant à votre tâche sur la map de l'écran d'accueil.
+- Soit en cliquant sur l'icône ![Liste de tâche](/assets/images/list_icon.png) en bas à droit de l'écran représentant la liste des tâches puis en cliquant sur la tâche désirée.
```
Comment (alexsegura) [inline fr/coursier/tache.md]: `à droit` → `à droite`

File: fr/coursier/preuve.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+lang: fr
+role: coursier
+title: Guide du coursier
```
Comment (alexsegura) [inline fr/coursier/preuve.md]: Title should be "Preuves de livraison" or something like this

File: fr/coursier/livraison.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+---
+lang: fr
+role: coursier
+title: Livraison
+ref: coursier-home
```
Comment (alexsegura) [inline fr/coursier/livraison.md]: `ref` should be the same for English & French versions (i.e the same as in file `en/courier/delivery.md`)

File: fr/coursier/preuve-livraison.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+---
+lang: fr
+role: coursier
+title: Preuve de livraison
+ref: coursier-home
```
Comment (alexsegura) [inline fr/coursier/preuve-livraison.md]: `ref` should be different, something like `courier-proof-of-delivery`

File: _data/sidebar.yml [hunk 7]
```diff
@@ -30,14 +30,17 @@ fr:
...
       url: /fr/administrateur/magasins.html
-    - title: Applications smartphone
-      url: /fr/administrateur/applications-smartphone.html
   restaurant:
       - title: Commandes
         url: /fr/restaurateur/commandes.html
   dev:
     - title: API
       url: /fr/developpeur/api.html
+  coursier:
```
Comment (alexsegura) [inline _data/sidebar.yml]: Should be `courier`

**Review by alexsegura — CHANGES_REQUESTED** [review]

**Comment by arnomasson** [comment]
je check ca

**Review by alexsegura — APPROVED** [review]

**Pull request merged**